state-chain/pallets/cf-flip/src/lib.rs

```diff
@@ -448,6 +450,29 @@ impl<T: Config> Bonding for Bonder<T> {
 			}
 		})
 	}
+
+	fn try_bond(authority: &Self::ValidatorId, bond: Self::Amount) -> DispatchResult {
+		Account::<T>::mutate_exists(authority, |maybe_account| {
+			if let Some(account) = maybe_account.as_mut() {
+				if account.balance >= bond {
+					account.bond = bond;
+					Ok(())
+				} else {
+					Err(Error::<T>::InsufficientLiquidity.into())
+				}
+			} else {
+				Err(Error::<T>::AccountDoesNotExist.into())
+			}
+		})
+	}
+
+	fn unbond(authority: &Self::ValidatorId) {
+		Account::<T>::mutate_exists(authority, |maybe_account| {
+			if let Some(account) = maybe_account.as_mut() {
+				account.bond = 0u128.into()
+			}
+		})
+	}
```

Comment:
I don't think we need to add two new methods here, we can just use existing methods. (see my other comments)